src/app/pages/products-list/card/card.component.ts

```diff
@@ -0,0 +1,19 @@
+import {Component} from '@angular/core';
+import {IProduct} from 'src/app/shared/products/product.interface';
+import {productMock} from '../../../shared/products/product.mock';
+
+@Component({
+    selector: 'app-card',
+    templateUrl: './card.component.html',
+    styleUrls: ['./card.component.css'],
+})
+export class CardComponent {
+    product: IProduct;
+    constructor() {
+        this.product = productMock;
+    }
```

Comment:
Почему не так решил сделать?
```suggestion
    product: IProduct = productMock;
```

Reply:
Изначально думал, что productMock буду получать через конструктор и в нём же записывать в переменную.
Исправил